### ISSUE TITLE
Add a template for creating a new LFE book project

### ISSUE THREAD

**Issue by oubiwann** [lfe-rebar3/rebar3_lfe#42]

**Comment by wmealing** [comment]
When you say lfe-book , do you mean like livebook ?

**Comment by diasbruno** [comment]
Maybe it would be nice to let this be another plugin.

**Comment by oubiwann** [comment]
@wmealing No, like what we're doing with the LFE Manual, using the CLI tool `mdbook`. 

@diasbruno Yeah, this has been in the backlog for a while, and I've never _really_ been 100% on it. For a while, I was experimenting a great deal with new books/boolets ... but the focus has shifted to something more maintainable: One Big Book.

I'll go ahead and just close this as "won't fix" ...
